src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -243,4 +255,37 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	protected toggleMoreActionsButton(isActiveOrHovered: boolean) {
 		this._cellToggleMoreActions.toggleVisible(!isActiveOrHovered);
 	}
+
+	public deltaDecorations(newDecorationRange: NotebookRange, oldDecorationRange: NotebookRange): void {
+		if (oldDecorationRange) {
+			this.removeDecoration(oldDecorationRange);
+		}
+
+		if (newDecorationRange) {
+			this.addDecoration(newDecorationRange);
+		}
+	}
+
+	private addDecoration(range: NotebookRange): void {
+		if (range && this.output && this.output.nativeElement) {
+			let hostElem = this.output.nativeElement;
+			let children = hostElem.children;
+			let ele = children[Math.ceil(range.lineNumber / 2)];
+			if (ele) {
+				ele.className = 'rangeHighlight';
```

Comment:
Using Dom.addClass and Dom.removeClass are much better. Currently this is going to overwrite any other classes set on this element which is probably not what you intend.